Assets/Scripts/StandardSamples/OpenID/OpenIdTokenManager.cs

```diff
@@ -28,6 +30,7 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Samples.OpenId
 {
+    using static PlayEveryWare.EpicOnlineServices.EOSManager;
```

Comment:
+1 for `using static`, I was previously unaware of or had forgotten its benefits, you make me write better code! 😃